models/errors/validation-errors.ts

```diff
@@ -8,10 +8,19 @@ export const messageTypeIssue: fhir.OperationOutcomeIssue = {
   expression: ["Bundle.entry.resource.ofType(MessageHeader).eventCoding.code"]
 }
 
-export const featureBlockedIssue: fhir.OperationOutcomeIssue = {
+export const incorrectScopeIssue: fhir.OperationOutcomeIssue = {
   severity: "fatal",
-  code: fhir.IssueCodes.NOT_SUPPORTED,
-  diagnostics: "The feature for this message type is disabled"
+  code: fhir.IssueCodes.FORBIDDEN,
```

Comment:
Wondering if we should return a 403 if the issue code is `FORBIDDEN`? What about `NOT_SUPPORTED` below?

Reply:
Made this change. Left `NOT_SUPPORTED` alone.